ihatemoney/currency_convertor.py

```diff
@@ -0,0 +1,48 @@
+import requests
+from diskcache import Cache
+
+
+class Singleton(type):
+    _instances = {}
+
+    def __call__(cls, *args, **kwargs):
+        if cls not in cls._instances:
+            cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
+        return cls._instances[cls]
+
+
+class CurrencyConverter(object, metaclass=Singleton):
+    # Get exchange rates
+    default = "No Currency"
+    api_url = "https://api.exchangeratesapi.io/latest?base=USD"
+    _internal_cache = Cache()
+
+    def __init__(self, expire=86400):
+        self.expire = expire
+
+    def set_rate(self):
+        if "rates" not in self._internal_cache:
+            rates = requests.get(self.api_url).json()["rates"]
+            rates[self.default] = 1.0
+            self._internal_cache.set("rates", rates, expire=self.expire)
+
+    def get_currencies(self):
+        self.set_rate()
+        rates = [rate for rate in self._internal_cache.get("rates")]
+        rates.sort()
```

Comment:
```suggestion
        rates.sort(key=lambda rate: "" if rate == self.default else rate)
```
This is done to force the 'No currency' to be the first on the list.

Reply:
^^ thanks! Added!